### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by Anandpskerala**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-hi.md [hunk 1]
```diff
@@ -1,24 +1,36 @@
-### Index
-
-* [C](#C)
-* [C++](#C++)
+## Index
```
Comment (SethiPandi) [inline free-courses-hi.md:1]: There should be 3 # symbols here.

File: free-courses-hi.md [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+* [C++](#c-1)
+* [HTML](#html)
+* [PHP](#php)
+* [Python](#python)
 
 ### C
 
 * [All C Concepts | Hindi](https://www.youtube.com/playlist?list=PL7ersPsTyYt1d8g5qaxbE6sjWDzs4D_1v) - Saurabh Shukla
 
-
```
Comment (SethiPandi) [inline free-courses-hi.md:11]: Add this line break back.

File: free-courses-hi.md [hunk 3]
```diff
@@ -1,24 +1,36 @@
...
+
+### Javascript
+
+* [JavaScript Tutorial In Hindi](https://www.youtube.com/watch?v=hKB-YGF14SY) - Harry
+
 
 ### Python
 
 * [Python Tutorial For Beginners In Hindi (With Notes)](https://www.youtube.com/watch?v=gfDE2a7MKjA) - Harry
+* [Python Basic in Hindi](https://masterprograming.com/python/)
```
Comment (SethiPandi) [inline free-courses-hi.md:31]: Add a line break here, there should be two.

**Review by SethiPandi — CHANGES_REQUESTED** [review]
You can read the contribution guidelines (CONTRIBUTING.md) for more information on formatting for the repository. Or click the failed check below and go to Travis CI to see exactly what was wrong with your PR and how to resolve the issue in future.

File: free-courses-hi.md [hunk 4]
```diff
@@ -1,24 +1,36 @@
...
```
Comment (SethiPandi) [inline free-courses-hi.md:30]: This isn't in alphabetical order, B is before T.

**Review by SethiPandi — CHANGES_REQUESTED** [review]
Noticed after the first review, sorry! Also isn't in alphabetical order.

**Comment by eshellman** [comment]
please resolve conflicts.

**Pull request closed without merging**